src/components/dictionaryConcepts/components/CreateMapping.jsx

```diff
@@ -8,19 +7,35 @@ import MapType from './MapType';
 class CreateMapping extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = { inputValue: this.props.to_concept_name || '', options: [], isVisible: false };
   }
 
   handleInputChange = (value) => {
     this.setState({ inputValue: value });
   }
 
+  handleKeyPress = (e, inputValue, url) => {
+    if (String(e.key).toUpperCase() === 'ENTER') {
```

Comment:
Same as above

Reply:
Yes at that point I convert the key entered to uppercase and check if its equivalent to `ENTER`

Reply:
We could listen for the keyCode instead,,, it's value for the enter key is usually 13

Reply:
Will this work for non English key boards?

Reply:
I have updated it by setting a constant `const KEY_CODE_FOR_ENTER = 13;` that I use to Identify the key code and also gave it a descriptive name for a new user